src/frequenz/sdk/timeseries/logical_meter/_logical_meter.py

```diff
@@ -29,36 +29,24 @@ class LogicalMeter:
         creating `LogicalMeter` instances.
 
     Example:
-        ```python
+        import asyncio
         from datetime import timedelta
 
         from frequenz.sdk import microgrid
-        from frequenz.sdk.timeseries import ResamplerConfig
+        from frequenz.sdk.actor import ResamplerConfig
+        from frequenz.client.microgrid import ComponentMetricId
 
         await microgrid.initialize(
-            "grpc://127.0.0.1:50051",
-            ResamplerConfig(resampling_period=timedelta(seconds=1))
+            "microgrid.sandbox.api.frequenz.io",
+            62060,
+            ResamplerConfig(resampling_period=timedelta(seconds=1)),
         )
 
-        logical_meter = microgrid.logical_meter()
-        pv_pool = microgrid.new_pv_pool(priority=5)
-        grid = microgrid.grid()
-
-        # Get a receiver for a builtin formula
-        pv_power_recv = pv_pool.power.new_receiver()
-        async for pv_power_sample in pv_power_recv:
-            print(pv_power_sample)
-
-        # or compose formulas to create a new formula
-        net_power_recv = (
-            (
-                grid.power - pv_pool.power
-            )
-            .build("net_power")
-            .new_receiver()
-        )
-        async for net_power_sample in net_power_recv:
-            print(net_power_sample)
+        logical_meter = microgrid.logical_meter().start_formula("#1001 + #1002", ComponentMetricId.ACTIVE_POWER)
+        logical_meter_receiver = logical_meter.new_receiver()
+
+        async for power in logical_meter_receiver:
+            print(power.value)       ```python
```

Comment:
It looks like this got messed up.

```suggestion
            print(power.value)
```